### ISSUE TITLE
--no-cli option is ignored

### ISSUE THREAD

**Issue by VolodyaIvanets** [coturn/coturn#503]
Hello!

I was trying to start turnserver version 4.5.1.1 with **no-cli** option but got error message:

```
# turnserver --no-cli
0: ERROR:
CONFIG ERROR: Empty cli-password, and so telnet cli interface is disabled! Please set a non empty cli-password!
```

**Comment by misi** [comment]
It will be fixed in next release..
See changelog:
https://github.com/coturn/coturn/blob/8eb779a3cf792804c5f5b9091857e88536ff3ba8/ChangeLog#L4

**Comment by misi** [comment]
fixed in 4.5.1.2
